core/app/modules/kit-library/assets/js/components/taxonomies-filter-list.js

```diff
@@ -1,16 +1,17 @@
+import { useState, useMemo } from 'react';
 import Taxonomy from '../models/taxonomy';
 import Collapse from './collapse';
 import SearchInput from './search-input';
 import { Checkbox, Text } from '@elementor/app-ui';
 import { sprintf } from '@wordpress/i18n';
-import { useState, useMemo } from 'react';
+import { eventTrackingDispatch } from 'elementor-app/event-track/events';
 
 const MIN_TAGS_LENGTH_FOR_SEARCH_INPUT = 15;
 
 const TaxonomiesFilterList = ( props ) => {
 	const [ isOpen, setIsOpen ] = useState( props.taxonomiesByType.isOpenByDefault );
 	const [ search, setSearch ] = useState( '' );
-
+	const category = ( '/' === props.category ? 'all kits' : 'favorites' );
```

Comment:
What if there is no catgory prop at all?

Reply:
then the category will favorites
changed the condition so that the fallback value is `all kits`